CMakeLists.txt

```diff
@@ -381,8 +381,16 @@ if(WINDOWS)
         )
     endif()
 endif()
+
 # set UMF_PROXY_LIB_ENABLED
-if(UMF_PROXY_LIB_BASED_ON_POOL STREQUAL SCALABLE)
+if(UMF_LINK_HWLOC_STATICALLY)
+    message(
```

Comment:
Why does proxy lib require HWLOC to be linked dynamically?

Reply:
I removed this check. I think something doesn't work in the past (problem with doubled symbols?) but now CI shows everything is OK